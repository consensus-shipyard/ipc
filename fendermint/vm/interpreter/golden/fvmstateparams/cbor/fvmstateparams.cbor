a86a73746174655f726f6f74d82a58250001bd3ffa3d1e11163d5fa40a156e3caef9cf45f736ce99b42abede4c32ccf9d50f9018966974696d657374616d701b146ef18ccc9e55bb6f6e6574776f726b5f76657273696f6e1568626173655f6665655100c167b71a6b28cec166ff4ca208fbb3446b636972635f737570706c795100fc8ea419cbcbc401a12180e388e68f5d68636861696e5f69641b000b6b07b4af45db6b706f7765725f7363616c65006b6170705f76657273696f6e01