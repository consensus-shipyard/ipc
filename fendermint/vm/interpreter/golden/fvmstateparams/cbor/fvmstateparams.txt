FvmStateParams { state_root: Cid(bag6t76r5dyirmpk7uqfbk3r4v3446rpxg3hjtnbkx3peymwm7hkq7eaysy), timestamp: Timestamp(1472379715227375035), network_version: NetworkVersion(21), base_fee: TokenAmount(257079523536971773801.541083398290518852), circ_supply: TokenAmount(335706089450661601774.571585084053688157), chain_id: 3213905584145883, power_scale: 0, app_version: 1 }